doc/source/guide/data_types/lightcurve.rst

```diff
@@ -116,7 +60,124 @@ the 'index' keyword, as is shown below.
-this has created a pandas DataFrame object with a column name "param1",
+this has created a pandas `~pandas.DataFrame` object with a column name "param1",
 with an index of times.
 
+3. Inspecting maps & Getting at the data
+----------------------------------------
+A lightcurve holds both data as well as meta data. The meta data for the lightcurve
+ is accessed by ::
+
+    >>> header = goes.meta
+
+This references the meta data dictionary with the header information as read
+from the source file. A word of caution, many data sources provide little to no
+meta data so this variable might be empty.
+
+The data in a SunPy `~sunpy.lightcurve.LightCurve` object is accessible through the
+`~sunpy.map.LightCurve.data` attribute.  The data is implemented as a
+Pandas `~pandas.DataFrame`, so to get a look at what data you have available ::
+
+    >>> goes.data
+
+You can also get a quick overview of what data you have available like so: ::
+
+    >>> goes.data.info()
+
+LightCurves are columnar data so to get at a particular datum you need to
+first index the column then the element you want. To get the names of the
+available columns: ::
+
+    >>> goes.data.columns
+
+So you can access the 0th element in the column `xrsa` with: ::
+
+    >>> goes.data['xrsa'][0]
+
+You can also grab all of the data at a particular time: ::
+
+    >>> goes.data['xrsa']['2013-07-21 23:59']
+
+This will return a list of entries with times that match the accuracy of the time
+you provide. Finally if you want to get at the x or y values: ::
+
+    >>> x = goes.data.index
+    >>> y = goes.data.values
+
+You can read more about indexing at the `pandas documentation website
+<http://pandas.pydata.org/pandas-docs/stable/>`_.
+
+3. Plotting
+-------------------------------------
+
+The SunPy LightCurve object has its own built-in plot methods so that
+it is easy to quickly view your lightcurve. To create a plot just
+type:
 
+.. plot::
+    :include-source:
+
+    from sunpy.lightcurve import GOESLightCurve
+    from sunpy.time import TimeRange
+    tr = TimeRange('2013/07/21', '2013/07/22')
+    goes = GOESLightCurve.create(tr)
+    goes.peek()
+
+This will open a matplotlib plot on your screen. The `~sunpy.lightcurve.LightCurve.peek()`
+function provides a custom view on the data while `~sunpy.lightcurve.LightCurve.plot()`
+provides a more generic plot.
+
+In addition, to enable users to modify the plot it is possible to grab the
+matplotlib axes object by using the `~sunpy.lightcurve.LightCurve.plot()` command.
+This makes it possible to use the SunPy plot as the foundation for a
+more complicated figure. For a bit more information about this and some
+examples see :ref:`plotting`. Here is one a more complicated example
+which makes use of this methodology.
+
+.. plot::
+    :include-source:
+
+    from sunpy.lightcurve import GOESLightCurve
+    from sunpy.time import TimeRange
+    tr = TimeRange('2013/07/21', '2013/07/22')
```

Comment:
There should be blank lines between the imports and the code in these examples to make them easier to read. Especially if the user downloads the file.

I am not going to block merging the PR over it, but we should try and do it in the future, and correct them as we find them.